### ISSUE TITLE
[package-python] error "ModuleNotFoundError: No module named 'packaging'"

### ISSUE THREAD

**Issue by ayush987goyal** [projen/projen#1747]
Getting the following error in a couple of PRs for a JSII enabled CDK construct library:

```
Error: Command (/tmp/npm-packF3vhd5/.env-SPYZ32/bin/python -m twine check /home/runner/work/cdk-monitoring-constructs/cdk-monitoring-constructs/.repo/dist/python/*) failed with status 1:
#STDERR> Traceback (most recent call last):
#STDERR>   File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/runpy.py", line 196, in _run_module_as_main
#STDERR>     return _run_code(code, main_globals, None,
#STDERR>   File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/runpy.py", line 86, in _run_code
#STDERR>     exec(code, run_globals)
#STDERR>   File "/tmp/npm-packF3vhd5/.env-SPYZ32/lib/python3.10/site-packages/twine/__main__.py", line 22, in <module>
#STDERR>     from twine import cli
#STDERR>   File "/tmp/npm-packF3vhd5/.env-SPYZ32/lib/python3.10/site-packages/twine/cli.py", line [18](https://github.com/cdklabs/cdk-monitoring-constructs/runs/5886631777?check_suite_focus=true#step:7:18), in <module>
#STDERR>     from packaging import requirements
#STDERR> ModuleNotFoundError: No module named 'packaging'
#STDERR> 
```

Example PRs:
1. https://github.com/cdklabs/cdk-monitoring-constructs/runs/5886631777?check_suite_focus=true
2. https://github.com/cdklabs/cdk-monitoring-constructs/runs/5886309571?check_suite_focus=true

Has anyone seen this issue with Python packaging?

**Comment by Chriscbr** [comment]
I believe this issue is fixed in jsii-pacmak v1.56.0. Can you check if this fixes the issue for your projects @ayush987goyal?

**Comment by ayush987goyal** [comment]
Yeah seems like it is resolved. Thanks @Chriscbr , will close this issue.
